dll/win32/imm32/imm.c

```diff
@@ -2808,21 +2808,33 @@ UINT WINAPI ImmGetVirtualKey(HWND hWnd)
 HKL WINAPI ImmInstallIMEA(
   LPCSTR lpszIMEFileName, LPCSTR lpszLayoutText)
 {
-    LPWSTR lpszwIMEFileName;
-    LPWSTR lpszwLayoutText;
-    HKL hkl;
+    INT cchFileName, cchLayoutText;
+    LPWSTR pszFileNameW, pszLayoutTextW;
+    HKL hKL;
 
-    TRACE ("(%s, %s)\n", debugstr_a(lpszIMEFileName),
-                         debugstr_a(lpszLayoutText));
+    TRACE("ImmInstallIMEA(%s, %s)\n", debugstr_a(lpszIMEFileName), debugstr_a(lpszLayoutText));
```

Comment:
This will output double function names with `DEBUGFORMAT=wine`:
https://reactos.org/wiki/Debugging#Configure_logging_format

Reply:
Human, which is better?

Reply:
I don't like Wine style. It isn't our standard.

Reply:
Deleted function name. OK.